apps/public-reference/src/forms/applications/FormBackLink.tsx

```diff
@@ -0,0 +1,17 @@
+import Link from "next/link"
+import { OnClientSide, t } from "@bloom-housing/ui-components"
+import ApplicationConductor from "../../../lib/ApplicationConductor"
+
+const FormBackLink = (props: { conductor: ApplicationConductor }) => (
+  <p className="form-card__back">
+    {OnClientSide() && props.conductor.config.steps.length > 1 && (
```

Comment:
How come `props.conductor.config.steps.length > 1` is the conditional here? Isn't steps the whole list, so if you're on the first step of a list of 5 steps, this would show the back button which is not what we want, right?

Could you just do `conductor.determinePreviousUrl() && ...`?

Reply:
Good call. :+1: